css/styles.css

```diff
@@ -0,0 +1,49 @@
+body {
+  background-color: #B3C3C0;
+  font-family: 'Nunito', sans-serif;
+}
+
+h1 {
+  text-align: center;
+  font-weight: bold;
+  font-size: 100px;
+  margin-bottom: 0px;
+  margin-top: 30px;
+}
+
+nav {
+  text-align: center;
+  font-style: italic;
+  font-size: 50px;
+}
+
+hr {
+  size: 100px;
+}
+
+div {
+  text-align: center;
+}
+
+.happy {
+}
+
+.intpagetext {
+  text-align: center;
+}
+
+.mainpic {
+  border: solid;
```

Comment:
when declaring borders please the thickness`1px` style `solid` then the color `black`. The color is optional as it will inherit from its parent container.